### ISSUE TITLE
Wrong ranking API?

### ISSUE THREAD

**Issue by dehenne** [immutable/gods-unchained-api#1]
is it possible that `https://api.godsunchained.com/v0/ranking` is `https://api.godsunchained.com/v0/rank` ?

**Comment by MustafaAnasKH99** [comment]
man its been two years since this has been reported. We need a moderator on this repo

**Comment by Tjorriemorrie** [comment]
I agree, these dead repos do not help. I would suggest they add someone from the community, but he won't know the code so it won't help.

**Comment by MustafaAnasKH99** [comment]
if the community is actually allowed to maintain this, it would be way better. you would be surprised what community members know about this api.
